app/apps/server/bridges/users.js

```diff
@@ -17,6 +22,33 @@ export class AppUserBridge {
 		return this.orch.getConverters().get('users').convertByUsername(username);
 	}
 
+	async create(user, appId) {
+		this.orch.debugLog(`The App ${ appId } is requesting to create a new user`);
+
+		user.password = 'pass';
```

Comment:
is there any purpose by having a fixed password? I'd recommend using a random password and having that password sent via email.
```suggestion
		user.password = Random.id();
```

Reply:
@sampaiodiego The idea was to have a fixed password, and when the bot logs in for the first time it has to change the password since `user.requirePasswordChange` is set to `true`. This way we don't have to send the mail to the bot email. @graywolf336  @sampaiodiego Your thoughts??